e2e/libs/backupstore/s3.py

```diff
@@ -24,70 +35,87 @@ def get_api_client(self, client, core_api, minio_secret_name):
 
         minio_endpoint_url = \
             base64.b64decode(base64_minio_endpoint_url).decode("utf-8")
-        minio_endpoint_url = minio_endpoint_url.replace('https://', '')
+        minio_endpoint_url = f"localhost:{self.PORT_FORWARD}"
+
+        minio_cert_file_path = "/e2e/minio_cert.crt"
```

Comment:
LGTM

I have only one question: I executed the PR locally without Docker. Should we update the [README.md](https://github.com/longhorn/longhorn-tests/blob/master/e2e/README.md) to include creating the e2e folder before executing the test cases?

Reply:
Yes, The `CodeFactor` check prevents us from using a temporary folder like `/tmp`, I can't find another solution but storing the cert file in the `/e2e` folder. If you come up with a better approach, feel free to refine it. Thank you!

Reply:
Refined by using current working directory. Thank you!